src/azure-cli-core/azure/cli/core/application.py

```diff
@@ -8,13 +8,13 @@
 import os
 import uuid
 import argparse
-from .parser import AzCliCommandParser
-from azure.cli._output import CommandResultItem
-import azure.cli.extensions
-import azure.cli._help as _help
-import azure.cli._logging as _logging
-from azure.cli._util import todict, CLIError
-from azure.cli._config import az_config
+from azure.cli.core.parser import AzCliCommandParser
+from azure.cli.core._output import CommandResultItem
+import azure.cli.core.extensions
```

Comment:
Will that PR closely follow this one or should we create an issue to track it?

---
In reply to: [77924059](https://github.com/Azure/azure-cli/pull/828#discussion_r77924059) [](ancestors = 77924059)